tester/tester.go

```diff
@@ -41,7 +41,9 @@ type Tester struct {
 	mClients sync.RWMutex
 	clients  map[string]*client
 
-	codecs      map[string]goka.Codec
+	mCodecs sync.RWMutex
+	codecs  map[string]goka.Codec
```

Comment:
idiomatic go would call it `codecsMu`

Reply:
Jop, but in goka all the Mutex are called like this:
```
mClients sync.RWMutex
mQueues     sync.Mutex
...
```
And i don't want to refactor the whole project, that's why i kept the naming schema and called it `mCodecs sync.RWMutex`